app/views/unattended/provisioning_templates/cloud_init/cloud_init_default.erb

```diff
@@ -55,10 +55,10 @@ runcmd:
 <% end -%>
 - |
 <%=
-  if puppet_enabled && (host_param_true?('enable-puppetlabs-repo') ||
-    host_param_true?('enable-puppetlabs-puppet6-repo') ||
-    host_param_true?('enable-puppetlabs-puppet5-repo'))
-    indent(2) { snippet 'puppetlabs_repo' }
+  if puppet_enabled && (host_param_true?('enable-puppet-official-repo') ||
+    host_param_true?('enable-puppet-official-puppet6-repo') ||
```

Comment:
I believe we need a migration of some sort to change the name of this parameter automatically if we rename this.

Reply:
That's what I suggested in https://github.com/theforeman/foreman/pull/8239#issuecomment-756839529 but I think an example of how to do so would be useful. I can't find a good example in the database migrations.

Reply:
@ekohl: Sorry, I missed your comment.
Basically, we want to migrate all parameter kinds. So a simple migration just needs something like this:

```ruby
Parameter.where(name: 'enable-puppetlabs-puppet6-repo').update_all(name: 'enable-puppet-official-puppet6-repo')
```

This goes into the `up` method. The `down` method is the same reversed.
Imho that should do the trick.
@kamils-iRonin: WDYT?

Reply:
@timogoebel at first glance, you're right